kotsadm/web/src/components/shared/FileTree.jsx

```diff
@@ -19,21 +19,21 @@ export default class FileTree extends React.Component {
   }
 
   getLevel = () => {
-    return this.props.level || 0
+    if (this.props.isSupportBundle) {
```

Comment:
how about passing the initial level? like
```
getLevel = () => {
  const initialLevel = this.props.initialLevel || 0;
  return this.props.level || initialLevel;
}
```
and then pass initialLevel from the SupportBundle component